reference/configuration.html.md.erb

```diff
@@ -206,6 +208,12 @@ As with the more verbose [`[[services]]`](#the-services-sections), you can speci
 
 * `internal_port` : The port this service (and application) will use to communicate with clients. The default is 8080. We recommend applications use the default.
 * `force_https`: A boolean which determines whether to enforce HTTP to HTTPS redirects.
+* `auto_stop_machines`: Whether to automatically stop instances of an application when there's excess capacity, per region. If there's only one instance in a region, then the instance is stopped if it has no traffic when the autostop process runs. The default it `true`.
+* `auto_start_machines`: Whether to automatically start instances of an application when a new request is made to the application and there's no excess capacity, per region. If there's only one instance in a region, then it's started whenever a request is made to the application. The default is `true`.
+
+<div class="callout">
+We recommend setting `auto_stop_machines` and `auto_start_machines` to the same value to avoid having instances that either never start or never stop. Learn more about [automatically starting and stopping Fly machines](/docs/apps/autostart-stop/), including how the process determines excess capacity in a region using the `soft_limit` setting.
+</div>
```

Comment:
Locally, the link within the callout was invisible (no underline). Is that normal?

Reply:
We need to ask Kyle for a style for links in a callout.